optuna/study/study.py

```diff
@@ -210,6 +210,9 @@ def trials(self) -> List[FrozenTrial]:
 
         Returns:
             A list of :class:`~optuna.trial.FrozenTrial` objects.
+            -- seealso::
+                See :func:`~optuna.study.Study.get_trails` for related method.
+
```

Comment:
Typo and fix syntax.
```suggestion

            .. seealso::
                See :func:`~optuna.study.Study.get_trials` for related method.

```